tardis/opacities/tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from copy import deepcopy
+from tardis.simulation import Simulation
+
+
+@pytest.fixture(scope="package")
```

Comment:
I find it is better to try and move fixtures to a higher conftest level if they are duplicates of other fixtures. Because this adds another simulation setup and run (with only 10 packets, I suppose) which has the same name. I am not sure if `package` scope is correct. Perhaps it should be `module`?

Reply:
I'll switch the scope to module, I was having some trouble finding the right fixtures so I opted to simply duplicate the old opacity_state tests here